lib/ChainwebDb/Types/Block.hs

```diff
@@ -1,59 +1,50 @@
-{-# OPTIONS_GHC -fno-warn-missing-signatures #-}
-{-# LANGUAGE DeriveAnyClass #-}
-{-# LANGUAGE DeriveGeneric #-}
-{-# LANGUAGE FlexibleContexts #-}
-{-# LANGUAGE FlexibleInstances #-}
-{-# LANGUAGE GADTs #-}
+{-# LANGUAGE DeriveAnyClass     #-}
+{-# LANGUAGE DeriveGeneric      #-}
+{-# LANGUAGE DerivingStrategies #-}
+{-# LANGUAGE FlexibleContexts   #-}
+{-# LANGUAGE FlexibleInstances  #-}
 {-# LANGUAGE ImpredicativeTypes #-}
-{-# LANGUAGE MultiParamTypeClasses #-}
-{-# LANGUAGE OverloadedStrings #-}
 {-# LANGUAGE StandaloneDeriving #-}
-{-# LANGUAGE TemplateHaskell #-}
-{-# LANGUAGE TypeApplications #-}
-{-# LANGUAGE TypeFamilies #-}
-{-# LANGUAGE TypeSynonymInstances #-}
-{-# LANGUAGE UndecidableInstances #-}
+{-# LANGUAGE TypeFamilies       #-}
+
+{-# OPTIONS_GHC -fno-warn-missing-signatures #-}
 
 module ChainwebDb.Types.Block where
 
+import Data.Aeson
+import Data.Word
+import Database.Beam
 ------------------------------------------------------------------------------
-import           Data.Aeson
-import           Data.Time
-import           Data.Word
-import           Database.Beam
-------------------------------------------------------------------------------
-import           ChainwebDb.Types.DbHash
-import           ChainwebDb.Types.Miner
-------------------------------------------------------------------------------
-
+import ChainwebDb.Types.DbHash
 
 ------------------------------------------------------------------------------
 data BlockT f = Block
-  { _block_id :: C f Int
-  , _block_creationTime :: C f UTCTime
-  , _block_chainId :: C f Int
-  , _block_height :: C f Int
-  , _block_hash :: C f DbHash
-  , _block_powHash :: C f DbHash
-  , _block_target :: C f DbHash
-  , _block_weight :: C f DbHash
-  , _block_epochStart :: C f UTCTime
-  , _block_nonce :: C f Word64
-  , _block_miner :: PrimaryKey MinerT f
-  } deriving Generic
+  { _block_id           :: C f Int
+  , _block_creationTime :: C f Int
+  , _block_chainId      :: C f Int
+  , _block_height       :: C f Int
+  , _block_hash         :: C f DbHash
+  -- , _block_powHash      :: C f DbHash
```

Comment:
I originally put this field in because this DB schema is supposed to be optimized for querying/analytics and I think the powHash is definitely something that people will want to do analysis on.  Is there any reason we can't put this in?